### ISSUE TITLE
Use of structure: multiple_files is required but not respected in folder collections

### ISSUE THREAD

**Issue by josineto** [sveltia/sveltia-cms#239]
In folder collections, to use multiple files for i18n (e.g. `post.en.md` and `post.pt.md`), usually one writes the following configuration:

```yaml
i18n:
  structure: multiple_files
  locales: [pt, en]
  default_locale: pt
```

But Sveltia requires also that one writes something like `path: '{{slug}}/index.{{locale}}'` in the folder configuration (in addition to above configuration), which seems to me to be unnecessary, because `.{{locale}}` should be implied from the use of `structure: multiple_files`. 

This was not always the case: I didn't use Sveltia the last months (for reasons not related to the CMS itself), and then when someday I tried to use it, all my collections showed empty in the CMS. Then I started to investigate version after version, and nailed it down: **this changed from** `v0.39.11` to `v0.39.12`.

I don't know if this is by design or a bug, but well, I've decided to submit this issue so just you know it. Thank you anyway, the project is going better with each version!

**Comment by kyoshino** [comment]
Thanks for flagging this. My local test can reproduce the issue. Fix coming.

- without `path`: works
- `path: '{{slug}}'`: works
- `path: '{{slug}}/index'`: fail

**Comment by kyoshino** [comment]
Fixed in [v0.45.2](https://github.com/sveltia/sveltia-cms/releases/tag/v0.45.2). You need to reload the CMS and revert `'{{slug}}/index.{{locale}}'` to `'{{slug}}/index'` to see the files. Sorry about that! 🙇🏼
